raiden-network/light-client
# dApp: hub proposal when selecting hub

Fixes #211 

**Short description**
This PR introduces suggested partners for the dApp when connecting to a token network.
<img width="650" alt="Screenshot 2020-12-11 at 17 44 44" src="https://user-images.githubusercontent.com/43838780/101930931-1323fe00-3bd9-11eb-856b-63dd0435a016.png">

<img width="636" alt="Screenshot 2020-12-11 at 18 08 44" src="https://user-images.githubusercontent.com/43838780/101933066-f3420980-3bdb-11eb-8c21-d4b438fe571e.png">

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Chose to connect to a new well-connected token (a list of three hubs should be displayed).
2. Chose to connect to a token with no partners (a message about not being able to fetch a list is displayed).
